File: SPRINGBOOT/demo/src/main/java/com/example/classes/Fattura.java
package com.example.classes;

public class Fattura {
    private int id;
    private double importo;
    private int idBuono;
    private String merce;
    private double peso;

    public Fattura(int id, double importo, int idBuono, String merce, double peso) {
        this.id = id;
        this.importo = importo;
        this.idBuono = idBuono;
        this.merce = merce;
        this.peso = peso;
    }

    public int getId() {
        return id;
    }

    public double getImporto() {
        return importo;
    }

    public int getIdBuono() {
        return idBuono;
    }

    public String getMerce() {
        return merce;
    }

    public double getPeso() {
        return peso;
    }
}

File: PHP/classi/Fattura.php
<?php

class Fattura {
    private $id;
    private $importo;
    private $idBuono;
    private $merce;
    private $peso;

    public function __construct($id, $importo, $idBuono, $merce, $peso) {
        $this->id = $id;
        $this->importo = $importo;
        $this->idBuono = $idBuono;
        $this->merce = $merce;
        $this->peso = $peso;
    }

    public function getId() {
        return $this->id;
    }

    public function getImporto() {
        return $this->importo;
    }

    public function getIdBuono() {
        return $this->idBuono;
    }

    public function getMerce() {
        return $this->merce;
    }

    public function getPeso() {
        return $this->peso;
    }
}
